### ISSUE TITLE
Cleanup Instruction table

### ISSUE THREAD

**Issue by chfast** [ethereum/cpp-ethereum#4072]
In files Instruction.(h|cpp), there are a lot of unnecessary data and unused functions.

**Comment by seusher** [comment]
It looks like this issue is resolved in EIP211: RETURNDATA #4062 (pending merge) and is no longer up-for-grabs.

**Comment by chfast** [comment]
No. #4062 has some changes for this, but everything. E.g. "side effect" is not used anywhere I think. So we should wait with starting this until #4062 is merged to avoid conflicts, but there is more to be done here.

**Comment by AnthonyBroadCrawford** [comment]
@chfast Is it okay if I run with this issue?  I'm new to the codebase and this seems like it may be a good place to start.  Let me know as I'm excited to contribute.

**Comment by chfast** [comment]
Yes, sure. It's yours.

Some things might have been improved recently, but check out the Instruction.* files for unused data.

**Comment by AnthonyBroadCrawford** [comment]
@chfast Awesome.  I’ll start to dig in this evening and hopefully have something to review sometime next week.  

Where’s the best place to find all coding standards, preferred syntax, etc?  Want to make sure once the PR is ready it’s in the best shape possible.  

Cheers.

**Comment by chfast** [comment]
See https://github.com/ethereum/cpp-ethereum/blob/develop/CODING_STYLE.md.

The best is to use clang-format to format your changes. In case big tables are not readable after reformatting with clang-format, you can come up with manual formatting by disabling clang-format it that section.

**Comment by chfast** [comment]
1. The `additional` field does not seem to be needed.
2. You can change the type of `name` from `std::string` to `const char* const`.
3. Changing `std::map` to flat array would also be nice, but probably should be done as an another PR.

**Comment by AnthonyBroadCrawford** [comment]
@chfast got it, helpful, thank you.

**Comment by AnthonyBroadCrawford** [comment]
@chfast I made the changes and opened this PR ( https://github.com/ethereum/cpp-ethereum/pull/4970 )

Please provide any/all feedback and I'll be happy to make the recommended changes.
